### PR TITLE
Fix Azure KeyVault on macOS 13.0 by providing the -sha1 parameter

### PR DESCRIPTION
**Pull request by v-mohithgc**
**Task name**: common-npm-packages\azure-arm-rest-v2

**Description**: Fix Azure KeyVault on macOS 13.0 by providing the -sha1 parameter

**Documentation changes required:** (Y/N) <Please mark if documentation changes are required>

**Added unit tests:** (Y/N) <Please mark if unit tests were added or updated according changes>

**Attached related issue:** (Y/N) <Please add link to related issue here>

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by PhilipsonJoseph — APPROVED** [review]

File: common-npm-packages/azure-arm-rest-v2/azure-arm-common.ts [hunk 1]
```diff
@@ -429,6 +429,7 @@ export class ApplicationTokenCredentials {
         var openSSLPath = tl.osType().match(/^Win/) ? tl.which(path.join(__dirname, 'openssl', 'openssl')) : tl.which('openssl');
         var openSSLArgsArray = [
             "x509",
+            "-sha1",
```
Comment (ozanovus) [inline common-npm-packages/azure-arm-rest-v2/azure-arm-common.ts:432]: I think this method is not used anymore, are you sure it is solving the issue?

File: common-npm-packages/azure-arm-rest-v2/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-azure-arm-rest-v2",
-  "version": "3.213.0",
+  "version": "3.216.0",
```
Comment (ozanovus) [inline common-npm-packages/azure-arm-rest-v2/package.json:3]: Hi, we're in sprint 215. Did you make it on purpose?

**Pull request merged**